satellite_populate/commands.py

```diff
@@ -23,28 +23,55 @@
 
 Use :code:`$ satellite-populate --help` for more info
 """
-import sys
-
 import click
+import os
+import sys
+import yaml
 
 from satellite_populate.constants import TEST_DATA
 from satellite_populate.main import populate
 
+CONFIG_FILE = '.satellite_populate.yaml'
+SATELLITE_POPULATE_FILE = 'SATELLITE_POPULATE_FILE'
+
+
+def _read_populate_settings(config_file):
+    """Parse satellite-populate configuration"""
+    if config_file.endswith(('.yml', '.yaml', 'json')):
+        with open(config_file) as file:
```

Comment:
'file' is an established keyword, can you choose something else? Perhaps 'conf' or  'config'